definition update, Constructors.md

ciphercoder opened the pull request:
Updated the definition #28086

## Summary

A minor update to the constructor definition, c#

Fixes #28086

ciphercoder committed:
definition update, Constructors.md

Updated the definition #28086

BillWagner commented on docs/csharp/programming-guide/classes-and-structs/constructors.md:
```diff
@@ -14,7 +14,7 @@ Whenever a [class](../../language-reference/keywords/class.md) or [struct](../..
 ## Constructor syntax
 
-A constructor is a method whose name is the same as the name of its type. Its method signature includes only the method name and its parameter list; it does not include a return type. The following example shows the constructor for a class named `Person`.
+A constructor is a method whose name is the same as the name of its type. Its method signature includes only an [access modifier](./access-modifiers.md), the method name and its parameter list; it does not include a return type. The following example shows the constructor for a class named `Person`.
```
The access modifier is optional (as is true for all methods). We should add that here:

```suggestion
A constructor is a method whose name is the same as the name of its type. Its method signature includes only an optional [access modifier](./access-modifiers.md), the method name and its parameter list; it does not include a return type. The following example shows the constructor for a class named `Person`.
```

BillWagner reviewed (commented):
Thanks for making this update @ciphercoder 

I had one minor change to suggest, and then this is ready to merge.

Thanks again for making a contribution to docs! 🎆

ciphercoder committed:
Update docs/csharp/programming-guide/classes-and-structs/constructors.md

I am delighted you accepted the update, yes I am incorporating the suggestion, 

Most Welcome!

Co-authored-by: Bill Wagner <wiwagn@microsoft.com>

BillWagner reviewed (approved):
Thanks again @ciphercoder 

I'll :shipit: as soon as the build finishes.
